_labo-help/similarity.markdown

```diff
@@ -14,7 +14,9 @@ This is the "Similarity" tool. In this brief help page you will:
 
 ## <a name="intro"></a>Intro
 
-In the Similarity tool you can watch a media resource, explore and navigate the content using visual and audio keyframes, and discover related keyframes from other media resoures. The tool enables researches to
+In the Similarity tool you can watch a media resource, explore and navigate the content using visual and audio keyframes. Moreover, it allows you to discover related keyframes from other media resoures. The tool enables researches to browse a part of the archive in a way that is complementary to the structured search offered in the Search tool. This may expose relationships that are not apparent from the metadata, emerging from the visual and audio domain. These relations are often more associative (like resembling composition or colours) than semantic (like picturing the same object or person). 
```

Comment:
Dit lijkt in de eerste instantie erg op de Resource Viewer. Misschien daaraan refereren om het verschil duidelijk te maken, of anders begin met de zin over het ontdekken van related keyframes